app/src/styles/StyleCardView.tsx

```diff
@@ -1,82 +1,79 @@
 import { styled } from '@mui/material/styles';
 import { Box, Typography } from '@mui/material';
 import ButtonLink from '@/components/ButtonLink';
-
-// Color Scheme:
-// #ECF2F8: Light blue
-// #96b6d4: Mild blue
-// #477096: Dark blue
-// #4F4846: Black
-// #FDF8F5: White
+import {LIGHT_BLUE, MILD_BLUE, DARK_BLUE, BLACK, WHITE } from '@/styles/ColorScheme';
```

Comment:
nit: there should be a space between `{ LIGHT_BLUE`. Do you need to configure prettier in your IDE?

Reply:
I do have prettier in my IDE, but sometimes I forget to save. I think I will need a habit of saving after writing new code or making changes in my codebase. I'll be more careful in the future.